includes/class-wc-cart-session.php

```diff
@@ -130,6 +130,11 @@ public function get_cart_from_session() {
 			}
 		}
 
+		// If it's not empty, it's been already populated by the loop above
```

Comment:
[Comments need to end in a full stop](https://travis-ci.org/woocommerce/woocommerce/jobs/416000791#L928)